includes/Admin/AnyCommentGenericSettings.php

```diff
@@ -820,21 +820,21 @@ public function init_settings () {
 				          ->checkbox()
 				          ->set_id( self::OPTION_NOTIFY_ON_NEW_COMMENT )
 				          ->set_title( __( 'Real-time updates', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Show comments in real-time. Users on the page would be notified about new comments by green alert and see comment without page reload.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Show new comments in real-time. Users would be notified about new comments by a green alert while the new comments load instantly on the page.', "anycomment" ) ) ),
 
 				     $this->field_builder()
 				          ->number()
 				          ->set_id( self::OPTION_INTERVAL_COMMENTS_CHECK )
 				          ->set_title( __( 'New Comment Interval Checking', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Interval (in seconds) to check for new comments. Minimum 5 and maximum is 100 seconds.', "anycomment" ) ) ),
+				          ->set_description( esc_html( __( 'Interval (in seconds) to check for new comments. The minimum is 5 and the maximum is 100.', "anycomment" ) ) ),
 
 
 				     // Admin email notifications
 				     $this->field_builder()
 				          ->checkbox()
 				          ->set_id( self::OPTION_NOTIFY_ADMINISTRATOR )
-				          ->set_title( __( 'Notify Administrator', "anycomment" ) )
-				          ->set_description( esc_html( __( 'Notify administrator via email about new comment.', "anycomment" ) ) ),
+				          ->set_title( __( 'Notify Administrators of new comments', "anycomment" ) )
```

Comment:
We have titles capitalized first letter, maybe we could make it the same way here, so that we won't need to fix it for many languages later

Reply:
```suggestion
				          ->set_title( __( 'Notify Administrators of New Comments', "anycomment" ) )
```